python_search/search_ui/fzf_kitty.py

```diff
@@ -10,17 +10,17 @@ class FzfInKitty:
     Renders the search ui using fzf + termite terminal
     """
 
-    FONT_SIZE = 15
+    FONT_SIZE : int = 15
     PREVIEW_PERCENTAGE_SIZE = 50
-    HEIGHT = 127
-    WIDTH = 950
-    RANK_TIE_BREAK = "begin,index"
+    _default_window_size = (800, 400)
```

Comment:
Make this upper case, variables that not change are also called constants and should be upper case.